### ISSUE TITLE
[va-language-toggle] aria-current using inproper syntax

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4545]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

`aria-current` for the current, selected language, is set to `language` which, is [not a valid aria-current prop](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current). 

## What I expected to happen

`aria-current` should be set to `true` on the selected language

## Steps to reproduce:

View source code for [va-language-toggle](https://design.va.gov/storybook/iframe.html?globals=&args=&id=components-va-language-toggle--default&viewMode=story&path=%2Fdocs%2Fcomponents-va-language-toggle--docs)
```
<a class="is-current-lang" href="https://design.va.gov/storybook/iframe.html?globals=&amp;args=&amp;id=components-va-language-toggle--default&amp;viewMode=story&amp;path=%2Fdocs%2Fcomponents-va-language-toggle--docs" hreflang="en" aria-current="language">English</a>
```

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [X] This bug is not blocking any work
- [ ] Other

## Acceptance criteria
- [ ] The currently selected language has the `aria-current="true"` prop on it.
